Formula/virtualenvwrapper.rb

```diff
@@ -62,14 +62,15 @@ class Virtualenvwrapper < Formula
   end
 
   def install
-    venv = virtualenv_create(libexec, "python3")
+    python3 = "python3.10"
+    venv = virtualenv_create(libexec, python3)
     venv.pip_install resources
     venv.pip_install buildpath
 
     bin.install_symlink libexec/"bin/virtualenvwrapper_lazy.sh"
     (bin/"virtualenvwrapper.sh").write <<~SH
       #!/bin/sh
-      export VIRTUALENVWRAPPER_PYTHON="#{libexec}/bin/python"
+      export VIRTUALENVWRAPPER_PYTHON="#{libexec}/bin/#{python3}"
```

Comment:
If it's already in `libexec` then we don't need to change it. It doesn't hurt, though.